### PR TITLE
Remove OpenBL.org, this service is not available anymore

### PR DESCRIPTION
**Pull request by altmas5**
OpenBL shutted down their service:
https://twitter.com/sshblorg/status/854669263671615489

Is making a false positive to the output of the script.

### CODE REVIEW

**Comment by altmas5** [comment]
[Waiting for pull request approval](http://devopsreactions.tumblr.com/post/49248447890/waiting-for-pull-request-approval)

Waiting

**Review by sananguliyev — APPROVED** [review]

**Pull request merged**
